README.md

```diff
@@ -326,3 +326,10 @@ texture when an image or video is clicked.
 scene](https://demo.shapespark.com/api-examples-texture-picker/#autoplay)
 
 
+## Minimap
+
+Set of utilities for handling minimap state. Need to be called after the scene has loaded.
+
+* `Viewer.isMinimapEnabled()` - returns `true` if minimap is enabled
+* `Viewer.enableMinimap()` - enable minimap, attach to viewer
```

Comment:
the doc rather uses third person - 'enables minimap and attaches it to the viewer', disables/removes below. (I'm not really sure if this should be 'viewer' or 'the viewer', but the doc already uses 'the viewer' so let's maybe use 'the')

Reply:
Thanks! Corrected